examples/agentInteractionsDemo/app.js

```diff
@@ -207,9 +209,17 @@ new Vue({
                                 .then((conv) => {
                                     let newConv = conv;
                                     newConv.customer = conv.participants.find((part) => {
-                                        return part.purpose === "customer";
+                                        return part.purpose === "customer" || part.purpose === "external";
                                     });
-                                    this.conversationsData.conversations.push(newConv);
+                                    if (newConv.customer.purpose === "external"){
```

Comment:
Technically, I think this logic would be best as `if (newConv.customer.externalContactId) {...}` since we could still have a linked contact with `purpose !== 'external'` - we'd just get the "more correct" name in this case.

Reply:
Ah, I totally missed this edge case. Thanks!